R/error-checks.R

```diff
@@ -85,6 +85,14 @@ validateIsPositive <- function(object, nullAllowed = FALSE) {
   }
 }
 
+validateIsNonNegative <- function(object, nullAllowed = FALSE) {

```

Comment:
Maybe call this ValidateIsStrictPositive?

Reply:
I think this function is redundant now, because of a later commit that removed the code where it was called.  We can remove it.  But 0 values are valid in this function, which is why 'non-negative' is suggested here (or maybe 'non strict positive'?)

Reply:
oh yes. The other one should be StrictPositive lol. At any rate, if we don't need it...let's forget it